src/Filters/Provider.php

```diff
@@ -56,7 +56,8 @@ public function getFiltersForQuery(ProductSearchQuery $query, int $idShop)
         if (empty($this->filters)) {
             $this->filters = $this->database->executeS(
             'SELECT type, id_value, filter_show_limit, filter_type FROM ' . _DB_PREFIX_ . 'layered_category
-            WHERE id_category = ' . (int) $query->getIdCategory() . '
+            WHERE controller = \'' . $query->getQueryType() . '\'
```

Comment:
```suggestion
            WHERE controller = "' . pSQL($query->getQueryType()) . '"
```

Reply:
`Single quotes should be used for string values like in the VALUES() list. Double quotes are supported by MySQL for string values as well, but single quotes are more widely accepted by other RDBMS, so it is a good habit to use single quotes instead of double.`